app/model/command/BatchTagCommand.scala

```diff
@@ -2,32 +2,38 @@ package model.command
 
 import model.command.CommandError._
 import model.jobs.JobHelper
-import play.api.libs.functional.syntax._
-import play.api.libs.json.{Format, JsPath}
+import org.cvogt.play.json.Jsonx
 import repositories._
 import services.Contexts
 
 import scala.concurrent.Future
 
-case class BatchTagCommand(contentIds: List[String], tagId: Long, operation: String) extends Command {
+case class BatchTagCommand(contentIds: List[String], toAddToTop: Option[Long], toAddToBottom: List[Long], toRemove: List[Long]) extends Command {
```

Comment:
Can `toAddToTop: Option[Long], toAddToBottom: List[Long], toRemove: List[Long]` be modelled as a single case class called `Operations`?